app/jobs/server_upload_job.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+# Allows backgrounding of server uploads, for when a ton of files are selected
+# to be uploaded.
+class ServerUploadJob < ApplicationJob
```

Comment:
We should add a comment saying what "server upload" means.

Reply:
Done.